Pargi/Base.lproj/Main.storyboard

```diff
@@ -310,24 +309,39 @@
                         <rect key="frame" x="0.0" y="0.0" width="375" height="389"/>
                         <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
                         <subviews>
-                            <mapView clipsSubviews="YES" multipleTouchEnabled="YES" contentMode="scaleToFill" mapType="standard" pitchEnabled="NO" showsUserLocation="YES" showsPointsOfInterest="NO" translatesAutoresizingMaskIntoConstraints="NO" id="kIJ-34-wPP">
+                            <mapView clipsSubviews="YES" multipleTouchEnabled="YES" contentMode="scaleToFill" mapType="standard" pitchEnabled="NO" showsUserLocation="YES" showsCompass="NO" showsPointsOfInterest="NO" translatesAutoresizingMaskIntoConstraints="NO" id="kIJ-34-wPP">
```

Comment:
Was this compass change on purpose? Isn't the compass in the top corner (i.e it would not interfere with the bottom button)?

Reply:
At first I put the button in the top and removed the compass because of that. Then I moved the button to the bottom and decided to remove the compass, as it kind of clashes with the design. I did however thought about actually animating the button image instead based on the heading. Not sure if that would work though, i.e. people would probably not consider it a "button" anymore. Would be a nice touch though.